### PR TITLE
Fixed build summary code breaking PR builds on Maven and Gradle.

### PR DESCRIPTION
**Pull request by HSAR**

### CODE REVIEW

File: Tasks/Gradle/task.json [hunk 1]
```diff
@@ -291,7 +291,8 @@
         "codeAnalysis_ToolFailed": "%s analysis failed.",
         "sqAnalysis_IncrementalMode": "Detected a PR build - running the SonarQube analysis in incremental mode",
         "sqAnalysis_BuildSummaryTitle": "SonarQube Analysis Report",
+        "sqAnalysis_BuildSummaryNotAvailableInPrBuild": "Detailed SonarQube reports are not available for pull request builds.",
         "sqAnalysis_TaskReportInvalid": "Invalid or missing task report. Check SonarQube finished successfully.",
-        "sqAnalysisBuildSummaryLine_OneProject": "Detailed SonarQube report"
+        "sqAnalysis_BuildSummary_LinkText": "Detailed SonarQube report"
```
Comment (HSAR) [inline Tasks/Gradle/task.json]: Changed this to match the other "sqAnalysis_BuildSummary..." strings.

**Comment by bgavrilMS** [comment]
LGTM

File: Tests/L0/Maven/_suite.ts [hunk 2]
```diff
@@ -935,6 +933,63 @@ describe('Maven Suite', function() {
             });
     });
 
+    it('Maven with SonarQube - Does not fails if report-task.txt is missing during a PR build', function(done) {
+        // Arrange
+        createTempDirsForSonarQubeTests();
+        var testSrcDir:string = __dirname;
+        var testStgDir:string = path.join(__dirname, '_temp');
+
+        // not a valid PR branch
```
Comment (HSAR) [inline Tests/L0/Maven/_suite.ts]: Incorrect comment - will change.

**Comment by HSAR** [comment]
No parallel test for Gradle - will add.

File: Tests/L0/Maven/_suite.ts (hunk 2 above)
Comment (HSAR) [inline Tests/L0/Maven/_suite.ts]: Resolved.

**Comment by HSAR** [comment]
Resolved.

**Pull request merged**